cumulus/client/block-builder/src/lib.rs

```diff
@@ -225,6 +218,8 @@ where
 	/// supplied by `self.api`, combined as [`BuiltBlock`].
 	/// The storage proof will be `Some(_)` when proof recording was enabled.
 	pub fn build(mut self) -> Result<BuiltBlock<Block, backend::StateBackendFor<B, Block>>, Error> {
+		self.execute_extrinsics()?;
```

Comment:
Why wasn't it here before?

Reply:
Becase previously the BlockBuilder executes the extrinsic when pushing it, see the removed `push` method.